### ISSUE TITLE
Version 1.14.4 - NugRunning Not Working

### ISSUE THREAD

**Issue by sthdsnts** [rgd87/NugRunning#57]
NugRunning Not Working in Classic WoW Era.

Client Version 1.14.4

**Comment by Yojione** [comment]
Agree, not working.  Since the WoW update 1.14.4 was supposed to harmonise code between wow versions, I did try NugRunning  from classic WotLK and Retail in Classic era... but none seemed to work (I didnt clear all option/config files.. just dropped in the LUA)

**Comment by d87** [comment]
Alright, updated version 10.1.2 should work, although without DRs

**Comment by Yojione** [comment]
Woohoo... thanks.. seems to be working (what are "DRs"?  do we need them?) .. 
Thank you so much... I am lost without this addon :)

**Comment by Yojione** [comment]
Doh.. I spoke to soon... yes, the bars are back, but the durations/timers are not..  They are important, so any chance you could take a look please :)

**Comment by d87** [comment]
Yeah, i'll fix it

**Comment by d87** [comment]
Alright, 10.1.3 should work

**Comment by Yojione** [comment]
Sorry... but still no timers on bars working :(

**Comment by Yojione** [comment]
Wohoo.. I see you have released a new version... 1.14.1 for game version 1.14.4
And pleased to say it works.. bars appear and they have correct labels and timers 
I have not fully tested... but main stuff (including grouping when you dot up more than one target) seems to be working great.
Does the new versioning mean you gave up on consolidating all versions into a single addon code base? and this is a classic-era special?
Anyway - your efforts are much appreciated.. THANKS a lot :)

**Comment by Yojione** [comment]
Oh, just noticed... the options for Nugrunning dont appear in the "addons" section of the "options" panel (and I do have options addon selected in my addons list)

**Comment by Yojione** [comment]
Hi, sorry, dont want to nag, but are you going to get a chance to try and fix the "no options tab" issue?  Please :)

**Comment by sthdsnts** [comment]
Thanks for fixing, can you close?

**Comment by sthdsnts** [comment]
> Hi, sorry, dont want to nag, but are you going to get a chance to try and fix the "no options tab" issue? Please :)

The developer doesn't test the addon?

**Comment by Yojione** [comment]
> 

Well.. others can post to say Im wrong... or he can ask (I dont know what might help him... there are no LUA errors)
But you gave my post a thumbs up (suggesting you agree with me?)... yet you asking for it to be closed .. and its been closed... means we cant expect a fix now?
So how do we get it fixed?

**Comment by sthdsnts** [comment]
Slavery

**Comment by d87** [comment]
I don't have a sub, PTR is down, there are no premades there anyway to test something other than immolate, none of you provide any errors, It worked when i tried it when PTR was up, including options

**Comment by d87** [comment]
Try doing this ```/script LoadAddOn('NugRunningOptions')```

**Comment by Yojione** [comment]
> Try doing this `/script LoadAddOn('NugRunningOptions')`. See if it helps

Thanks.. and sorry I cant provide you errors.. but like I said, there are no LUA errors to give you.
I can provide screen shots.. but it would be a pretty boring screen shot... it just does not show in the addons tab.
I did run the script... and it added the title "NugRunning" in the addons list.. but if I selected, it showed no options.. 
![WoWScrnShot_090723_095111](https://github.com/rgd87/NugRunning/assets/88988907/4b348519-d1b5-4cc8-891d-e76626777238)
(hope I have attached a screen shot for you)

**Comment by d87** [comment]
Try 1.14.2 and if still nothing type
```
/console scriptErrors 1
/reload
```
And then see if any errors pop up

**Comment by Yojione** [comment]
Gave it a quick try, and whatever you did, seems to have worked.. 
Bars working and showing timers.. 
And now the options are showing in the addons tab... and the options seem to work (just tested the move anchor... but the spell lists seems to be there and default to warlock OK)
So thanks again so much for your work.. really appreciate it all working again :)
